src/keys.js

```diff
@@ -330,15 +332,19 @@ export function validateExtendedPublicKey(xpubString, network) {
  * - Must be a valid BIP32 public key.
  *
  * @param {string} pubkeyHex - (compressed) public key in hex
+ * @param {string} addressType - (optional) one of P2SH, P2SH-P2WSH, P2WSH
```

Comment:
Sorry, I should've been more explicit. There's a specific notation in jsdocs (which is the formatting being used here) for indicating a param is optional (see [docs](https://jsdoc.app/tags-param.html#optional-parameters-and-default-values)). The reason this is important is because we use these to generate [documentation](https://unchained-capital.github.io/unchained-bitcoin/) which is generated based on these comments.  

Here's the sample from the [jsdoc documentation](https://jsdoc.app/tags-param.html#optional-parameters-and-default-values):
```
@param {string} [somebody] - Somebody's name.
```

Reply:
fixed @bucko13! Thank for pointing it out